src/storm-pars-cli/storm-pars.cpp

```diff
@@ -366,10 +371,19 @@ void parameterSpacePartitioningWithSparseEngine(std::shared_ptr<storm::models::s
 
     auto engine = rvs.getRegionCheckEngine();
     STORM_PRINT_AND_LOG(" using " << engine);
+    auto splittingStrategy = rvs.getRegionSplittingStrategy();
+    auto estimateKind = rvs.getRegionSplittingEstimateMethod();
+    // TODO doesn't link
+    // STORM_PRINT_AND_LOG(" and splitting strategy " << splittingStrategy);
```

Comment:
```suggestion
    // STORM_PRINT_AND_LOG(" and splitting strategy " << splittingStrategy << "\n");
```
I don't see why this should not work, maybe just a matter of not re-running cmake?

Reply:
That may be, I'll take a look